core/jobs.py

```diff
@@ -932,7 +941,10 @@ def validate(cls, unused_mapper_spec):
 
 
 class BaseMapReduceJobManagerForContinuousComputations(BaseMapReduceJobManager):
-
+    """Continuous Computation jobs which runs continuously, are used to perform
+    statistical, visualisation or any other real time calculations. These jobs
+    are triggered by events for which a particular job is subscribed to.
```

Comment:
What do you mean when you say the job is "triggered by an event"? I thought these jobs run continuously in the background...

Reply:
updated.